### ISSUE TITLE
Suggestion: add homebrew releases

### ISSUE THREAD

**Issue by Hisu0** [binogure-studio/city-game-studio-i18n#238]
A new type of release, probably with a checkbox next to size?

IRL, homebrew is essentially a non-commercial, unofficial project that isn't supposed to be a money-maker, but can be something that the platform owners would never allow on it for some reason.

In game, I suggest the following scheme:
Pros:
- a homebrew release gets ratings regardless of its platform/audience and platform/genre compatibility; basically, it's an automatic perfect match in those two aspects.
- homebrew releases can be made without buying a platform license
- having a homebrew release on your portfolio (even if it's not on sale anymore) gives a minor boost to hype and slows down hype decay

Cons:
- you cannot set the price for a homebrew release; instead, the game sets it at random, but not higher than 35%
- not higher than 25% if you didn't buy a license
- having a homebrew release on your portfolio decreases publishing contract payouts and increases all license prices

So basically, homebrews will be a way to get fans rather than money, at the cost of dedicating yourself to self-publishing.

**Comment by xsellier** [comment]
Interesting idea really. Having another way to get more fans is nice, however you can already get a lot of fans by making trade publishing games. Those are like normal games but:
- You don't have to manage marketing
- You don't earn a lot of money (at most 10% of the normal income)
- The publisher is getting richer
- You get a lot of fans

This piece of gameplay has a huge downside, your competitors are getting more powerful, which is, for late game really important. With homebrew releases you won't get a real downside for it unless I misunderstood it

**Comment by Hisu0** [comment]
There are three major downsides of making homebrews:
1. The price a homebrew project sells for is not just less lucrative upfront, but also random: you can make 25% tops, but that can just as easily be 1%, you have no control over it.
2. Platforms become more expensive the more you do homebrews — but only for you, not for your competitors.
3. Also, the more you do homebrews, all ways to make money except for self-publishing become less profitable — again, just for you, not your competitors.

**Comment by xsellier** [comment]
If you don't mind I have several questions:
- Since homebrew are not commercial, why do you want to publish one?
- Homebrew will be available for old platforms only?
- Can homebrew be replaced by fangames? (let's say you've made a hit on Wataggi 2600, but you never published a sequel. Then 10 years laters, you see a fangame based on your game and you have to make a choice: Sue them (d*** move), hire them (valve move), buy them (EA move)

**Comment by Hisu0** [comment]
- It's not _really_ publishing, it's just making a release, setting the product live, but as you don't have different release mechanics aside from publishing, that'll have to do, I suppose. Creating an entire new gameplay mechanic for a single feature seems excessive for me.
- Homebrew is available for all platforms, but whenever you release a homebrew, all platforms you haven't licensed yet, both existing and future ones, go up in price a little to reflect your reputation among the platform holders.
- They _are_ essentially fan games, but rather than one-person hobby thing, they are still full-fledged studio projects, its just the studio openly refuses to go through platform holder's approval procedures (like Bible Games by Wisdom Tree on NES). So while I definitely like the idea of a fan game random event, it's not that.

BTW, there's also Bethesda move: support fans unofficially, making all future projects a little bit more costly, but have fans cover some of your games' flaws (basically, penalty to costs, bonus to leniency when factoring in scores).

**Comment by xsellier** [comment]
This suggestions is a really good because I have an idea about it.

What about, being able to create a community around your company. Once you have hired at least one marketing manager (Or maybe I will add a community manager not sure yet) you can start working on community task such as:
- Build a community
- Fund/support a homebrew project
- _Lent_ an employee for a homebrew project
- Craft merchandizing stuff for your community

**Upper side:**
- More fans
- Might be able to hire perfect employees

**Down side:**
- Cost a lot
- Might generate a new competitor

This would be a whole new gameplay mechanic and I like it. If you have more idea about it or if you don't like it please tell me.

**Comment by Hisu0** [comment]
That's more in the direction of a fan game rather than building a dedicated homebrew, but I'd like to see yours as well. Or, maybe, do both? ^_^

**Comment by xsellier** [comment]
I think I'm gonna implement that, and the API will allow you to implement the other way to do homebrew :-) (thanks to modding support)
